### PR TITLE
Refine song logs

### PR DESCRIPTION
**Pull request by veronicz**
see commit messages

### CODE REVIEW

File: sound-bubble/server/main.js [hunk 1]
```diff
@@ -5,6 +5,17 @@ import './methods/accountMethods';
 import './methods/groupMethods';
 import './publication';
 
+SyncedCron.add({
+  name: 'Update rencently played for all users',
```
Comment (martipal) [inline sound-bubble/server/main.js]: nit: typo

File: sound-bubble/imports/ui/components/Account/UserFeed.jsx [hunk 2]
```diff
@@ -7,11 +7,24 @@ import { withTracker } from 'meteor/react-meteor-data';
...
+const DEFAULT_LIMIT = 20;
+const LIMIT_INCREMENT = 20;
+const limit = new ReactiveVar(DEFAULT_LIMIT);
+
 class UserFeed extends Component {
   componentDidMount() {
     this.props.fetchMySongLogs();
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
```
Comment (martipal) [inline sound-bubble/imports/ui/components/Account/UserFeed.jsx]: is nextState still needed here?

File: sound-bubble/imports/ui/components/Account/UserFeed.jsx [hunk 3]
```diff
@@ -7,11 +7,20 @@ import { withTracker } from 'meteor/react-meteor-data';
...
 
+const DEFAULT_LIMIT = 20;
+const LIMIT_INCREMENT = 20;
+const limit = new ReactiveVar(DEFAULT_LIMIT);
+
 class UserFeed extends Component {
-  componentDidMount() {
-    this.props.fetchMySongLogs();
+  shouldComponentUpdate(nextProps, nextState) {
+    return nextProps.myRecentTracksReady;
```
Comment (martipal) [inline sound-bubble/imports/ui/components/Account/UserFeed.jsx]: Is nextState still needed?

File: sound-bubble/imports/ui/components/Account/UserFeed.jsx [hunk 4]
```diff
@@ -35,7 +44,9 @@ class UserFeed extends Component {
           <ul>{this.getSongDetails()}</ul>
         </div>
         <div className="show_more_button_container">
-          <button className="btn btn-secondary btn-sm"> Show More </button>
+          <button className="btn btn-secondary btn-sm" onClick={this.loadMore}>
+            Show More
+          </button>
```
Comment (Talos6) [inline sound-bubble/imports/ui/components/Account/UserFeed.jsx]: So are we doing show more button or just scroll down?

Comment (veronicz) [reply sound-bubble/imports/ui/components/Account/UserFeed.jsx]: Which one do you prefer? I haven't implemented scroll down yet cuz we haven't reached a consensus.

File: sound-bubble/imports/ui/components/Account/UserFeed.jsx (hunk 3 above)
Comment (veronicz) [reply sound-bubble/imports/ui/components/Account/UserFeed.jsx]: removed

**Pull request merged**